classes/system/stacklight/server/cluster.yml

```diff
@@ -7,3 +8,10 @@ classes:
 parameters:
   _param:
     kibana_elasticsearch_host: ${_param:monitor_vip_address}
+  keepalived:
+    cluster:
+      instance:
+        VIP:
```

Comment:
The problem is that VIP is defined in the formula. So if I renamed it I will have two different configuration.